### DISCUSSION TITLE
Issue with / question about domain verification

### DISCUSSION THREAD

**Discussion by verduijnt** [vercel/community#1795] [Help]
### Summary

Hello fellow Vercel users,

I've got an issue with the verification of my domain.
I registered my domain terryficsoftware.com at TransIP (a Dutch registrar)
I've done the same for another domain, which works perfectly.

However, the domain terryficsoftware.com didn't show up in the domains in my Vercel account.
I double checked the name servers and everything is the same as the other domain I successfully manage through Vercel.
When I created the project for terryficsoftware and added the domains, I got the message that I needed to verify my domain:
"Domain is pending verification"
I needed to add a TXT to _vercel with a verification code.
In the DNS settings of my domain in TransIP I added the TXT with name _vercel and set the verification code as value. TTL of 1 min.

However, I'm almost a day further and it still hasn't been resolved.

I have contacted TransIP but they say that be cause the nameservers are set to ns1.vercel-dns.com and ns2.vercel-dns.com I need to change DNS settings at Vercel. But the domain isn't added in the domains list, so I can't edit or add any DNS settings.

Also, when browsing to terryficsoftware.com I get some kind of Polish website. My guess it that that website is hosted on Vercel and someone has added the domain terryficsoftware.com as one of their domains.

So my questions are:

- Where do I enter this verification code?
- Is there something else I can do / check on my end?
- How is it possible that a domain name that I own can be used to display a totally different website?

Any help is much appreciated, really banging my head about this.

**Comment by amyegan** [comment]
Hi Terry. It looks like there is a problem with your TXT record. Can you try removing it and adding it again to be sure all the values are correct?
